src/mono/mono/mini/mini-wasm-debugger.c

```diff
@@ -1240,6 +1288,23 @@ describe_variables_on_frame (MonoStackFrameInfo *info, MonoContext *ctx, gpointe
 	return TRUE;
 }
 
+EMSCRIPTEN_KEEPALIVE void
+mono_wasm_get_deref_ptr_value (void *value_addr, MonoClass *klass)
+{
+	MonoType *type = m_class_get_byval_arg (klass);
+	if (type->type != MONO_TYPE_PTR && type->type != MONO_TYPE_FNPTR) {
+		DEBUG_PRINTF (2, "BUG: mono_wasm_get_deref_ptr_value: Expected to get a ptr type, but got 0x%x\n", type->type);
+		return;
+	}
+
+	if (!value_addr)
+		return;
+
+	gpointer val = *(gpointer**) value_addr;
+	mono_wasm_add_properties_var ("deref", -1);
```

Comment:
Note that might crash the app. It might be better to do this in js (access the ArrayBuffer object which represents the memory of the wasm instance).